### PR TITLE
new icon: jetbrains (plain)

### LINKED ISSUE #128: JetBrains icons out of date
All the JetBrains product icons appear to be the versions prior to the modern branding update the occurred. Could these all be updated to the modern branding set that JetBrains uses?

See https://www.jetbrains.com/company/press/index.html#product-logos for all current product icons.

### PR DESCRIPTION
**Pull request by amino19**
# **New icon section**
Added a new icon for JetBrains plain SVG, by fixing the current plain. The updated JetBrains icon has a missing `s` in the logo i.e., corrected.

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [ ] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date

---

# **New feature section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *New Feature: a brief description of feature*

## Notes
- Take reference from the attached issue.

### CODE REVIEW

**Comment by amino19** [comment]
What's wrong with the image?
It's showing both having `128`x`128`, then it's inherited automatically, & not predicting.

image

File: icons/jetbrains/jetbrains-plain.svg [hunk 1]
```diff
@@ -1 +1,14 @@
...
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN" "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="body_1" width="128" height="128">
```
Comment (amacado) [inline icons/jetbrains/jetbrains-plain.svg:3]: You have set the `width` and `height`, but the `viewBox="0 0 128 128` is missing in the `<svg>`. See https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/viewBox for more information about `viewBox`

File: icons/jetbrains/jetbrains-plain.svg [hunk 2]
```diff
@@ -1 +1,14 @@
...
+</g>
+</svg>
```
Comment (Panquesito7) [inline icons/jetbrains/jetbrains-plain.svg:14]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><path d="M0 38.467c0 5.648 3.664 10.418 8.74 12.135l.114.038.343.115 44.458 13.547c.305.115.649.191.992.191a3.16 3.16 0 0 0 3.168-3.167c0-1.145-.61-2.138-1.527-2.71L21.065 28.697a12.638 12.638 0 0 0-8.28-3.053C5.724 25.644 0 31.407 0 38.468v-.001z" fill="#FFF"/><path d="M49.076 1.488 5.99 27.628A12.781 12.781 0 0 0 0 38.468c0 7.06 5.725 12.822 12.784 12.822 3.244 0 6.182-1.221 8.434-3.206l38.58-31.445c.115-.076.192-.152.306-.267 1.985-1.717 3.244-4.236 3.244-7.06A9.3 9.3 0 0 0 54.037 0c-1.794 0-3.512.572-4.961 1.488z" fill="#FFF"/><path d="M44.802 9.311a9.245 9.245 0 0 0 3.243 7.06l.115.115 69.377 58.463a6.242 6.242 0 0 0 4.045 1.45c3.473 0 6.259-2.824 6.259-6.296a6.379 6.379 0 0 0-1.87-4.504L60.677 2.748C58.997 1.068 56.67 0 54.113 0a9.3 9.3 0 0 0-9.311 9.311z" fill="#FFF"/><path d="m117.92 65.066-79.033 38.58a12.731 12.731 0 0 0-6.984 11.411c0 7.06 5.724 12.784 12.784 12.784 2.786 0 5.343-.916 7.442-2.404l73.08-50.144c1.64-1.145 2.708-3.015 2.708-5.152 0-3.473-2.823-6.297-6.258-6.297-1.45 0-2.71.458-3.74 1.222z" fill="#FFF"/><path d="M89.489 38.352H38.352V89.49H89.49V38.352z" fill="#010101"/><path d="M63.12 79.91H43.923v3.206h19.195V79.91zM43.121 51.556l1.413-1.335c.381.458.763.725 1.26.725.533 0 .915-.382.915-1.107v-5.038h2.214v5.076c0 .992-.268 1.718-.802 2.251-.535.535-1.26.802-2.175.802-1.412-.038-2.252-.61-2.825-1.374zm6.755-6.793h6.449v1.87H52.09v1.222h3.816v1.755H52.09v1.26h4.274v1.869h-6.488v-7.975zm9.578 1.947h-2.403v-1.946h6.983v1.945H61.67v6.03h-2.214v-6.03zM44 55.906h3.778c.992 0 1.717.267 2.175.725.306.344.497.763.497 1.298v.038c0 .916-.535 1.488-1.222 1.793.954.305 1.603.878 1.603 1.947v.038c0 1.374-1.107 2.213-3.015 2.213h-3.778v-8.051H44zm4.275 2.443c0-.458-.344-.687-.993-.687h-1.107v1.412h1.069c.649 0 1.03-.23 1.03-.725zm-.764 2.289h-1.373v1.488h1.373c.687 0 1.07-.267 1.07-.725v-.038c.037-.458-.306-.725-1.07-.725zm4.007-4.732h3.549c1.221 0 2.06.305 2.595.84.458.458.725 1.068.725 1.87v.038c0 1.22-.648 2.022-1.64 2.48l1.908 2.786h-2.558l-1.603-2.404h-.725v2.404h-2.214v-8.013h-.037zm3.473 3.816c.725 0 1.183-.381 1.183-.954v-.038c0-.649-.458-.954-1.183-.954h-1.222v1.946h1.222z" fill="#FFF"/><path d="M61.44 55.83h2.137l3.396 8.053h-2.365l-.573-1.45h-3.091l-.573 1.45h-2.29l3.359-8.053zm1.946 4.885-.877-2.252-.916 2.252h1.793zm6.144-4.847h-2.213v7.976h2.213v-7.976zm.84 0h2.06l2.9 4.236v-4.236h2.176v7.976H75.56l-2.977-4.35v4.35H70.37v-7.976zm7.517 6.793 1.222-1.488c.801.61 1.679.954 2.595.954.61 0 .916-.19.916-.534v-.039c0-.343-.267-.496-1.374-.763-1.717-.381-3.015-.878-3.015-2.48v-.038c0-1.489 1.183-2.557 3.091-2.557 1.374 0 2.404.381 3.282 1.068l-1.107 1.565c-.724-.534-1.526-.801-2.251-.801-.534 0-.802.229-.802.496v.038c0 .382.268.534 1.374.763 1.832.382 2.977.992 2.977 2.48v.039c0 1.64-1.298 2.595-3.244 2.595-1.336.038-2.671-.42-3.664-1.298z" fill="#FFF"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: icons/jetbrains/jetbrains-plain.svg (hunk 2 above)
Comment (amino19) [reply icons/jetbrains/jetbrains-plain.svg]: @Panquesito7: Do you suggesting me a tool for resizing in the same manner `devicon` reqd.?

Comment (Panquesito7) [reply icons/jetbrains/jetbrains-plain.svg]: > @Panquesito7: Do you suggesting me a tool for resizing in the same manner `devicon` reqd.?

[`iloveimg`](https://www.iloveimg.com/resize-image/resize-svg) is a very good tool for resizing SVG icons. You just have to learn a bit how to use it well. 🙂

File: icons/jetbrains/jetbrains-plain.svg [hunk 3]
```diff
@@ -1 +1 @@
...
...947v.038c0 1.374-1.107 2.213-3.015 2.213h-3.778v-8.051H44zm4.275 2.443c0-.458-.344-.687-.993-.687h-1.107v1.412h1.069c.649 0 1.03-.23 1.03-.725zm-.764 2.289h-1.373v1.488h1.373c.687 0 1.07-.267 1.07-.725v-.038c.037-.458-.306-.725-1.07-.725zm4.007-4.732h3.549c1.221 0 2.06.305 2.595.84.458.458.725 1.068.725 1.87v.038c0 1.22-.648 2.022-1.64 2.48l1.908 2.786h-2.558l-1.603-2.404h-.725v2.404h-2.214v-8.013h-.037zm3.473 3.816c.725 0 1.183-.381 1.183-.954v-.038c0-.649-.458-.954-1.183-.954h-1.222v1.946h1.222z" fill="#FFF"/><path d="M61.44 55.83h2.137l3.396 8.053h-2.365l-.573-1.45h-3.091l-.573 1.45h-2.29l3.359-8.053zm1.946 4.885-.877-2.252-.916 2.252h1.793zm6.144-4.847h-2.213v7.976h2.213v-7.976zm.84 0h2.06l2.9 4.236v-4.236h2.176v7.976H75.56l-2.977-4.35v4.35H70.37v-7.976zm7.517 6.793 1.222-1.488c.801.61 1.679.954 2.595.954.61 0 .916-.19.916-.534v-.039c0-.343-.267-.496-1.374-.763-1.717-.381-3.015-.878-3.015-2.48v-.038c0-1.489 1.183-2.557 3.091-2.557 1.374 0 2.404.381 3.282 1.068l-1.107 1.565c-.724-.534-1.526-.801-2.251-.801-.534 0-.802.229-.802.496v.038c0 .382.268.534 1.374.763 1.832.382 2.977.992 2.977 2.48v.039c0 1.64-1.298 2.595-3.244 2.595-1.336.038-2.671-.42-3.664-1.298z" fill="#FFF"/></svg>
```
Comment (Panquesito7) [inline icons/jetbrains/jetbrains-plain.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128"><path d="M0 38.467c0 5.648 3.664 10.418 8.74 12.135l.114.038.343.115 44.458 13.547c.305.115.649.191.992.191a3.16 3.16 0 0 0 3.168-3.167c0-1.145-.61-2.138-1.527-2.71L21.065 28.697a12.638 12.638 0 0 0-8.28-3.053C5.724 25.644 0 31.407 0 38.468v-.001z"/><path d="M49.076 1.488 5.99 27.628A12.781 12.781 0 0 0 0 38.468c0 7.06 5.725 12.822 12.784 12.822 3.244 0 6.182-1.221 8.434-3.206l38.58-31.445c.115-.076.192-.152.306-.267 1.985-1.717 3.244-4.236 3.244-7.06A9.3 9.3 0 0 0 54.037 0c-1.794 0-3.512.572-4.961 1.488z"/><path d="M44.802 9.311a9.245 9.245 0 0 0 3.243 7.06l.115.115 69.377 58.463a6.242 6.242 0 0 0 4.045 1.45c3.473 0 6.259-2.824 6.259-6.296a6.379 6.379 0 0 0-1.87-4.504L60.677 2.748C58.997 1.068 56.67 0 54.113 0a9.3 9.3 0 0 0-9.311 9.311z"/><path d="m117.92 65.066-79.033 38.58a12.731 12.731 0 0 0-6.984 11.411c0 7.06 5.724 12.784 12.784 12.784 2.786 0 5.343-.916 7.442-2.404l73.08-50.144c1.64-1.145 2.708-3.015 2.708-5.152 0-3.473-2.823-6.297-6.258-6.297-1.45 0-2.71.458-3.74 1.222z"/><path d="M89.489 38.352H38.352V89.49H89.49V38.352z" fill="#010101"/><path d="M63.12 79.91H43.923v3.206h19.195V79.91zM43.121 51.556l1.413-1.335c.381.458.763.725 1.26.725.533 0 .915-.382.915-1.107v-5.038h2.214v5.076c0 .992-.268 1.718-.802 2.251-.535.535-1.26.802-2.175.802-1.412-.038-2.252-.61-2.825-1.374zm6.755-6.793h6.449v1.87H52.09v1.222h3.816v1.755H52.09v1.26h4.274v1.869h-6.488v-7.975zm9.578 1.947h-2.403v-1.946h6.983v1.945H61.67v6.03h-2.214v-6.03zM44 55.906h3.778c.992 0 1.717.267 2.175.725.306.344.497.763.497 1.298v.038c0 .916-.535 1.488-1.222 1.793.954.305 1.603.878 1.603 1.947v.038c0 1.374-1.107 2.213-3.015 2.213h-3.778v-8.051H44zm4.275 2.443c0-.458-.344-.687-.993-.687h-1.107v1.412h1.069c.649 0 1.03-.23 1.03-.725zm-.764 2.289h-1.373v1.488h1.373c.687 0 1.07-.267 1.07-.725v-.038c.037-.458-.306-.725-1.07-.725zm4.007-4.732h3.549c1.221 0 2.06.305 2.595.84.458.458.725 1.068.725 1.87v.038c0 1.22-.648 2.022-1.64 2.48l1.908 2.786h-2.558l-1.603-2.404h-.725v2.404h-2.214v-8.013h-.037zm3.473 3.816c.725 0 1.183-.381 1.183-.954v-.038c0-.649-.458-.954-1.183-.954h-1.222v1.946h1.222z" fill="#FFF"/><path d="M61.44 55.83h2.137l3.396 8.053h-2.365l-.573-1.45h-3.091l-.573 1.45h-2.29l3.359-8.053zm1.946 4.885-.877-2.252-.916 2.252h1.793zm6.144-4.847h-2.213v7.976h2.213v-7.976zm.84 0h2.06l2.9 4.236v-4.236h2.176v7.976H75.56l-2.977-4.35v4.35H70.37v-7.976zm7.517 6.793 1.222-1.488c.801.61 1.679.954 2.595.954.61 0 .916-.19.916-.534v-.039c0-.343-.267-.496-1.374-.763-1.717-.381-3.015-.878-3.015-2.48v-.038c0-1.489 1.183-2.557 3.091-2.557 1.374 0 2.404.381 3.282 1.068l-1.107 1.565c-.724-.534-1.526-.801-2.251-.801-.534 0-.802.229-.802.496v.038c0 .382.268.534 1.374.763 1.832.382 2.977.992 2.977 2.48v.039c0 1.64-1.298 2.595-3.244 2.595-1.336.038-2.671-.42-3.664-1.298z" fill="#FFF"/></svg>
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍
@MattIPv4, would you like to check if all is fine? Thanks. 🙂

File: icons/jetbrains/jetbrains-plain.svg (hunk 2 above)
Comment (amino19) [reply icons/jetbrains/jetbrains-plain.svg]: `iloveimg` consider everything under `%`, how to set height & width in `px`?

`devicon` consider everything in `px`, ig! :thinking:

Comment (amino19) [reply icons/jetbrains/jetbrains-plain.svg]: Just learnt, this won't happen next time for sure! :+1:

**Comment by Panquesito7** [comment]
Could you please make the text transparent so it looks fine in Icomoon? Let us know if you need any help! Thanks. 😄

**Comment by amino19** [comment]
Actually Pan, it's a [brand asset of JetBrains](https://www.jetbrains.com/company/brand/#report-a-violation), as I shared it in `devicon` discord's server.
I hope you understand the thing. :slightly_smiling_face: 

Also, it would be interesting editing the `.svg` file with transparent text!

**Comment by Panquesito7** [comment]
> Actually Pan, it's a [brand asset of JetBrains](https://www.jetbrains.com/company/brand/#report-a-violation), as I shared it in `devicon` discord's server. I hope you understand the thing. 🙂
> 
> Also, it would be interesting editing the `.svg` file with transparent text!

Yep, but the text cannot be seen, and we need to make the text transparent so it looks good. 🙂

**Comment by Thomas-Boi** [comment]
Hey @amino19,

Perhaps [this](https://www.youtube.com/watch?v=Men1K2oMq8g) might help.

File: icons/jetbrains/jetbrains-plain.svg [hunk 4]
```diff
@@ -1 +1 @@
...
```
Comment (maltejur) [inline icons/jetbrains/jetbrains-plain.svg:1]: This should cut out the text from the middle
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128" xml:space="preserve"><path d="M48.6 61.4c0-.5-.3-.8-1.1-.8h-1.4v1.5h1.4c.7 0 1.1-.2 1.1-.7zM61.6 60.7h1.8l-.9-2.2zM48.3 58.3c0-.5-.3-.7-1-.7h-1.1V59h1.1c.6.1 1-.2 1-.7zM55 57.8h-1.2v1.9H55c.7 0 1.2-.4 1.2-1s-.5-.9-1.2-.9z"/><path d="M125.9 65.5 60.7 2.7C59 1.1 56.7 0 54.1 0H54c-.4 0-.7 0-1 .1h-.3c-1.4.2-2.6.7-3.7 1.5L6.7 27.2c-1.2.6-2.3 1.5-3.2 2.5l-.2.2c-.3.3-.5.6-.8.9-.1.1-.2.2-.2.3-.2.4-.5.7-.7 1.1 0 .1-.1.1-.1.1-.2.4-.4.9-.6 1.3 0 .1-.1.2-.1.3-.1.3-.2.7-.3 1.1 0 .2-.1.3-.1.5-.1.3-.1.7-.2 1 0 .2-.1.3-.1.5-.1.5-.1 1-.1 1.5 0 .4 0 .8.1 1.2v.4c0 .3.1.5.1.8 0 .1.1.3.1.4.1.2.1.5.2.7 0 .1.1.3.1.4.1.2.2.5.3.7.1.3.2.6.4.9.1.1.1.3.2.4.1.2.2.4.4.7 0 .1.1.2.1.2.2.2.3.5.5.7 1.7 2.3 4.1 4 7 4.8l28.9 8.8v29.9h29.5l-29 14.2c-4.3 2.2-7 6.6-7 11.4 0 7.1 5.7 12.8 12.8 12.8 2.8 0 5.3-.9 7.4-2.4l73.1-50.1c1.6-1.1 2.7-3 2.7-5.2 0-1.9-.8-3.5-2-4.7zM38.4 43.4l-5.6-4.7L54 21.4l20.1 17H38.4v5zm21.1 3.3h-2.4v-1.9h7v1.9h-2.4v6h-2.2v-6zM44 55.9h3.8c1 0 1.7.3 2.2.7.3.3.5.8.5 1.3 0 .9-.5 1.5-1.2 1.8 1 .3 1.6.9 1.6 1.9 0 1.4-1.1 2.2-3 2.2H44v-7.9zm5.9-3.2v-8h6.4v1.9h-4.2v1.2h3.8v1.8h-3.8v1.3h4.3v1.9h-6.5zm-6.8-1.1 1.4-1.3c.4.5.8.7 1.3.7s.9-.4.9-1.1v-5h2.2V50c0 1-.3 1.7-.8 2.3-.5.5-1.3.8-2.2.8-1.4-.2-2.2-.8-2.8-1.5zm20 31.5H43.9v-3.2h19.2v3.2zm-7-19.2-1.6-2.4h-.7v2.4h-2.2v-8h3.5c1.2 0 2.1.3 2.6.8s.7 1.1.7 1.9c0 1.2-.6 2-1.6 2.5l1.5 2.3c.4-.9 3.2-7.6 3.2-7.6h2.1l3.4 8.1h-2.4l-.6-1.5h-3.1l-.6 1.5m9.2-.1h-2.2v-8h2.2v8zm8 0h-1.9l-3-4.3v4.3h-2.2v-8h2.1l2.9 4.2v-4.2h2.2v8zm7.3-2.4c0 1.6-1.3 2.6-3.2 2.6-1.3 0-2.7-.4-3.7-1.3l1.2-1.5c.8.6 1.7 1 2.6 1 .6 0 .9-.2.9-.5s-.3-.5-1.4-.8c-1.7-.4-3-.9-3-2.5 0-1.5 1.2-2.6 3.1-2.6 1.4 0 2.4.4 3.3 1.1l-1.1 1.6c-.7-.5-1.5-.8-2.3-.8-.5 0-.8.2-.8.5 0 .4.3.5 1.4.8 1.9.2 3 .8 3 2.4zm4.7 17.5V51.3l20.8 17.5-20.8 10.1z"/></svg>
```

**Comment by maltejur** [comment]
Also, the `original` and `plain` are now sized differently.

image
image

Maybe the `original` should be resized to fit the whole viewbox.

**Comment by amino19** [comment]
@Thomas-Boi: Shall you give me a short brief, what to change as now discussed?

**Comment by Panquesito7** [comment]
> @Thomas-Boi: Shall you give me a short brief, what to change as now discussed?

I think you need to

1. Make the text transparent.
2. Fix the size's as seen in https://github.com/devicons/devicon/pull/899#issuecomment-959752911
3. Maybe apply https://github.com/devicons/devicon/pull/899#discussion_r742167166

**Comment by maltejur** [comment]
3. is for 1. btw :D

File: icons/jetbrains/jetbrains-plain.svg [hunk 5]
```diff
@@ -1 +1,9 @@
...
...ATRAq0MiM7wKQGjVJO51zu5nEC6AZlpTiPCwrRWhKwXBHzXMdADKCTgXOE15E5jyXAX9qCroTbLMNoCWECqxlxncB8GJGwdyJKc91AMggQgXW8rDvAtBVNbHNNoAOMacCDTGvolCqCoY75jzXASDj6FSgoXAsfcpzGUjeyjbbc74LAZB9hAqs50HfBSAxVQVzJw54rgNAjjD8gTUxBJJb4865fb6LAJA/dCqwJoZAcmdOwZkdBAoAiSBUoJl7fReAWIyLbbYBJIzhDzRlZkckDfquA22pKVjZMeO5DgAFQKcCURz0XQDaMiXpSgIFgG6hU4FIzOy4pLLvOhBJTcGZHVOe6wBQMHQqEBXdimyYUdCdmPJcB4AColOByJhbkWo1BRtZjXuuA0CB0alAK+hWpNOMgpUd457rAFBwdCrQEjObljTsuw6cdZBdMQGkBaECLTGziqRZ33VAVQWTMaue6wCAsxj+QEvCixjDIH6tHAJW9V0IANSjU4G2mNmspIrvOgpmTsFGVlXPdQBAQ3Qq0K7dClYcoDvGFUzGrHquAwDWRKcCbTOzUUkTvuvIuTkFcydmPNcBAE3RqUDbnHOTkiY9l5FnU+IQMAAZQqcCHWN+RexqYpttABlEpwJxGFGwxBGdmxLbbAPIKDoViIWZDSrYv6Lst5LMqknaFw4pAUAmESoQm3BjrGkRLFo1o2C4Y85zHQDQEYY/EJtwueOIWGoaVU1Bd2KEQAEgD+hUIHbhUMhhMXlzPVWxzTaAnKFTgdiF77qZvLk2ttkGkEt0KpAoM5uQNOq7jpSoiu4EgByjU4FEOed2S9rnu44UGJc0QqAAkGd0KtAV4cqQw5IG/VbSdXNim20ABUGnAl0RvkMfUvCOvSjGxTbbAAqETgW6zsyGJY0pv6tD5kR3AgCA7jGzvWZ23PLjuJnt9f26AoAvdCrglZmVJe2VdLeyuxNnTdJnJI0752p+SwEAoODMrGxmByxbnYvjFtRc9v36AQCABsxs1MxmfaaFJmbNbNT36wQAacPwB1LLgmWod0raIf9LUecUHEt+L3tNAEBjhApkQhgwhiXdGv63G2YkPShphiABAM0RKpBJFixLrUjapqCLMdzhQ84o6EY8KanKclAAaB2hArkSho0Vw2vcrKpXj2evsmIDAOLx/wMpKeQ1ZqTHOAAAAABJRU5ErkJggg=="/>
```
Comment (Panquesito7) [inline icons/jetbrains/jetbrains-plain.svg:7]: Looks like you're adding a PNG file in an SVG file. That won't work properly. 😅

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: icons/jetbrains/jetbrains-plain.svg (hunk 5 above)
Comment (amino19) [reply icons/jetbrains/jetbrains-plain.svg:7]: @JOS-RE: Pl head over to this! :eyes:

File: icons/jetbrains/jetbrains-plain.svg [hunk 6]
```diff
@@ -1 +1,9 @@
...
+</defs>
+</svg>
```
Comment (Panquesito7) [inline icons/jetbrains/jetbrains-plain.svg:9]: Just apply this suggestion and all should be good (from @maltejur's suggestion). 🙂
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128" xml:space="preserve"><path d="M48.6 61.4c0-.5-.3-.8-1.1-.8h-1.4v1.5h1.4c.7 0 1.1-.2 1.1-.7zm13-.7h1.8l-.9-2.2zm-13.3-2.4c0-.5-.3-.7-1-.7h-1.1V59h1.1c.6.1 1-.2 1-.7zm6.7-.5h-1.2v1.9H55c.7 0 1.2-.4 1.2-1s-.5-.9-1.2-.9z"/><path d="M125.9 65.5 60.7 2.7C59 1.1 56.7 0 54.1 0H54c-.4 0-.7 0-1 .1h-.3c-1.4.2-2.6.7-3.7 1.5L6.7 27.2c-1.2.6-2.3 1.5-3.2 2.5l-.2.2c-.3.3-.5.6-.8.9-.1.1-.2.2-.2.3-.2.4-.5.7-.7 1.1 0 .1-.1.1-.1.1-.2.4-.4.9-.6 1.3 0 .1-.1.2-.1.3-.1.3-.2.7-.3 1.1 0 .2-.1.3-.1.5-.1.3-.1.7-.2 1 0 .2-.1.3-.1.5-.1.5-.1 1-.1 1.5 0 .4 0 .8.1 1.2v.4c0 .3.1.5.1.8 0 .1.1.3.1.4.1.2.1.5.2.7 0 .1.1.3.1.4.1.2.2.5.3.7.1.3.2.6.4.9.1.1.1.3.2.4.1.2.2.4.4.7 0 .1.1.2.1.2.2.2.3.5.5.7 1.7 2.3 4.1 4 7 4.8l28.9 8.8v29.9h29.5l-29 14.2c-4.3 2.2-7 6.6-7 11.4 0 7.1 5.7 12.8 12.8 12.8 2.8 0 5.3-.9 7.4-2.4l73.1-50.1c1.6-1.1 2.7-3 2.7-5.2 0-1.9-.8-3.5-2-4.7zM38.4 43.4l-5.6-4.7L54 21.4l20.1 17H38.4v5zm21.1 3.3h-2.4v-1.9h7v1.9h-2.4v6h-2.2v-6zM44 55.9h3.8c1 0 1.7.3 2.2.7.3.3.5.8.5 1.3 0 .9-.5 1.5-1.2 1.8 1 .3 1.6.9 1.6 1.9 0 1.4-1.1 2.2-3 2.2H44v-7.9zm5.9-3.2v-8h6.4v1.9h-4.2v1.2h3.8v1.8h-3.8v1.3h4.3v1.9h-6.5zm-6.8-1.1 1.4-1.3c.4.5.8.7 1.3.7s.9-.4.9-1.1v-5h2.2V50c0 1-.3 1.7-.8 2.3-.5.5-1.3.8-2.2.8-1.4-.2-2.2-.8-2.8-1.5zm20 31.5H43.9v-3.2h19.2v3.2zm-7-19.2-1.6-2.4h-.7v2.4h-2.2v-8h3.5c1.2 0 2.1.3 2.6.8s.7 1.1.7 1.9c0 1.2-.6 2-1.6 2.5l1.5 2.3c.4-.9 3.2-7.6 3.2-7.6h2.1l3.4 8.1h-2.4l-.6-1.5h-3.1l-.6 1.5m9.2-.1h-2.2v-8h2.2v8zm8 0h-1.9l-3-4.3v4.3h-2.2v-8h2.1l2.9 4.2v-4.2h2.2v8zm7.3-2.4c0 1.6-1.3 2.6-3.2 2.6-1.3 0-2.7-.4-3.7-1.3l1.2-1.5c.8.6 1.7 1 2.6 1 .6 0 .9-.2.9-.5s-.3-.5-1.4-.8c-1.7-.4-3-.9-3-2.5 0-1.5 1.2-2.6 3.1-2.6 1.4 0 2.4.4 3.3 1.1l-1.1 1.6c-.7-.5-1.5-.8-2.3-.8-.5 0-.8.2-.8.5 0 .4.3.5 1.4.8 1.9.2 3 .8 3 2.4zm4.7 17.5V51.3l20.8 17.5-20.8 10.1z"/></svg>

```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thanks for your contribution! 😄👍
@MattIPv4, feel free to check if all is good. 🙂

**Review by Thomas-Boi — APPROVED** [review]
This looks great @amino19! I'll merge it now

**Pull request merged**